app/mixins/preload_data.js

```diff
@@ -18,6 +18,9 @@ export default Ember.Mixin.create({
     }
 
     if (this.get("session.authToken")) {
+      promises = promises.concat(
```

Comment:
shouldn't this be outside of the `if (this.get("session.authToken"))` condition ?

What happens if you load the app, and then log-in ? Won't those records be missing ?